animationHelpers.js

```diff
@@ -81,9 +73,9 @@ function animate(duration = 1000) {
         [xTransVal, yTransVal, zTransVal] = interpolatedTranslation[currentStep];
 
         renderOnce();
-
+        vertexArray = getFrameData();
         currentStep++;
```

Comment:
!suggest All instances of vertexArray assignments should be wrapped inside getVertices() function. For example this one should be: vertexArray = getVertices(getFrameData());